public/usage-examples/graphics/draw_circle_on_bitmap/draw_circle_on_bitmap-1-red-planet-oop.cs

```diff
@@ -0,0 +1,36 @@
+using SplashKitSDK;
+
+namespace DrawCircleOnBitmap
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a bitmap to draw on
+            Bitmap planet = new Bitmap("planet", 400, 400);
+            
+            // Fill background with dark color
+            planet.ClearBitmap(Color.Black);
+            
+            // Create color
+            Color red = Color.Red;
+            
+            // Draw the main planet circle
+            planet.FillCircleOnBitmap(Color.RGBAColor(180, 0, 0, 255), 200, 200, 150);
+            planet.DrawCircleOnBitmap(red, 200, 200, 150);
+            
+            // Add some surface details with smaller circles
+            for(int i = 0; i < 15; i++)
+            {
+                double x = Rnd(100, 300)  // Random between 100 and 300
+                double y = Rnd(100, 300)   // Random between 100 and 300
+                double size = Rnd(10, 30)  // Random between 10 and 30
+                planet.DrawCircleOnBitmap(red, x, y, size);
+            }
+            
+            // Save and free the bitmap
+            planet.SaveBitmap("draw_circle_on_bitmap-1-red-planet");
+            planet.FreeBitmap();
+        }
+    }
+}
```

Comment:
This entire program doesnt run as there is numerous errors.
OOP code needs to use `using SplashKitSDK;` then each time a splashkit function is called you do `SplashKit.ClearBitmap`.

Line 13 and line 16 aren't calling Splashkit colour correctly which causes an error.

Lines 25, 26 and 27 all are missing ; which causes an error there. I've worked through troubleshooting this code and this is the corrected version - 

```csharp
using SplashKitSDK;

namespace DrawCircleOnBitmap
{
    public class Program
    {
        public static void Main()
        {
            // Create a bitmap to draw on
            Bitmap planet = new Bitmap("planet", 400, 400);
            
            // Fill background with dark color
            SplashKit.ClearBitmap(planet, SplashKit.ColorBlack());
            
            // Create color
            Color red = SplashKit.ColorRed();
            
            // Draw the main planet circle
            SplashKit.FillCircleOnBitmap(planet, SplashKit.RGBAColor(180, 0, 0, 255), 200, 200, 150);
            SplashKit.DrawCircleOnBitmap(planet, red, 200, 200, 150);
            
            // Add some surface details with smaller circles
            for(int i = 0; i < 15; i++)
            {
                double x = SplashKit.Rnd(100, 300);  // Random between 100 and 300
                double y = SplashKit.Rnd(100, 300);  // Random between 100 and 300
                double size = SplashKit.Rnd(10, 30);  // Random between 10 and 30
                SplashKit.DrawCircleOnBitmap(planet, red, x, y, size);
            }
            
            // Save and free the bitmap
            SplashKit.SaveBitmap(planet, "draw_circle_on_bitmap-1-red-planet");
            SplashKit.FreeBitmap(planet);
        }
    }
}
```

Please ensure you are reviewing and checking these program codes correctly run prior to doing pull requests.